scripts/openpose.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=image_recognition_openpose node
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+User=amigo
+Restart=on-abort
+ExecStart=/bin/bash -c 'source /home/amigo/.tue/setup.bash && export PATH=/usr/local/cuda/bin:$PATH && export LD_LIBRARY_PATH=/usr/local/cuda/lib64$:$LD_LIBRARY_PATH && rosrun image_recognition_openpose openpose_node _net_input_width:=368 _net_input_height:=368 _net_output_width:=368 _net_output_height:=368 _model_folder:=/home/amigo/openpose/models/ __ns:=hero/openpose image:=/hero/head_rgbd_sensor/rgb/image_raw'
```

Comment:
Is there a way to make this more readable?

Reply:
I don’t think so. Maybe multi-line would be possible.

Reply:
multi line is possible using a backslash \

Reply:
Indeed, testing is required

Reply:
Works